src/Centreon/Infrastructure/HostConfiguration/API/Model/HostGroupV21.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * Copyright 2005 - 2021 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Infrastructure\HostConfiguration\API\Model;
+
+/**
+ * This class is designed to represent the formatted response of the API request.
+ *
+ * @package Centreon\Infrastructure\HostConfiguration\API\Model
+ */
+class HostGroupV21
+{
+
+    /**
+     * @var int|null
+     */
+    public $id;
+
+    /**
+     * @var string|null
+     */
+    public $name;
```

Comment:
Did the property of the Model should be in adequation with your src/Centreon/Domain/HostConfiguration/Model/HostGroup ? If so those two properties don't match.